tests/kubectl.py

```diff
@@ -20,12 +20,18 @@
 
 def launch(command, ok_to_fail=False, ns=namespace, timeout=60):
     # Build command
-    cmd = f"{kubectl_cmd}"
+    cmd = f"{kubectl_cmd} "
+    cmd_args = command.split(" ")
     if ns is not None and ns != "" and ns != "--all-namespaces":
-        cmd += f" --namespace={ns}"
+        cmd += f"{cmd_args[0]} --namespace={ns} "
```

Comment:
latest version of kubectl doesn't allow `kubectl --namespace=XXX command` format, only `kubectl command --namespace=XXX`